src/components/QuoteRequest/Groomer/GroomerSentRequests.jsx

```
@@ -6,8 +6,9 @@ import { Schedule, Corgi, Note } from "/public/Icons";
 function GroomerSentRequests({ Infos }) {
   return (
     <>
-      <GroomerEstimate Info={Infos[0]} />
-      <GroomerEstimate Info={Infos[1]} />
+      {Infos.map((Info) => {
+        return <GroomerEstimate Info={Info} />;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)